flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/RemoveCachedShuffleDescriptorTest.java

```diff
@@ -294,42 +324,7 @@ private void testRemoveCacheForPointwiseEdgeAfterFailover(
         assertEquals(expectedAfter, blobWriter.numberOfBlobs());
     }
 
-    private DefaultScheduler createSchedulerAndDeploy(
```

Comment:
It's better to keep this method for the convenience of tests in this test class. It can just invoke the `createSchedulerAndDeploy` in the util class though.